components/event-publisher/org.wso2.carbon.event.publisher.admin/src/main/java/org/wso2/carbon/event/publisher/admin/EventPublisherAdminService.java

```diff
@@ -456,7 +456,19 @@ public boolean deployWSO2EventPublisherConfiguration(String eventPublisherName,
         return true;
     }
 
+    @Deprecated
     public boolean deployTextEventPublisherConfiguration(String eventPublisherName,
+                                                          String streamNameWithVersion,
+                                                          String eventAdapterType,
+                                                          String textData,
+                                                          BasicOutputAdapterPropertyDto[] outputPropertyConfiguration,
+                                                          String dataFrom, boolean mappingEnabled)
+            throws AxisFault {
+        return deployCacheableTextEventPublisherConfiguration(eventPublisherName, streamNameWithVersion, eventAdapterType,
+                textData, outputPropertyConfiguration, dataFrom, 15, mappingEnabled);
```

Comment:
@lgobinath Since we are setting a default cache timeout value of 15 everywhere for the deprecated methods, shall we introduce a constant for the value 15 and name it accordingly for better readability?